package.json

```diff
@@ -45,11 +54,11 @@
     "split": "^1.0.0"
   },
   "scripts": {
-    "test": "./bin/run-test.sh",
-    "system-test": "./bin/run-system-test.sh",
-    "changelog": "./bin/run-changelog.sh",
-    "coverage": "./bin/run-test.sh -c",
-    "bump": "./bin/run-bump.sh",
-    "closure": "./node_modules/.bin/closure-npc"
+    "test": "gulp test",
```

Comment:
The master branch has the same differences.  That is, local testing doesn't do coverage, but coverage is done on travis.  Since this PR is consistent with master in terms of behavior, I think addressing this difference can be done after the transition to typescript is complete.